egov-indexer/pgr-migration-batch-indexer.yml

```
@@ -17,20 +17,20 @@ ServiceMaps:
               - inJsonPath: $
                 outJsonPath: $.Data
             externalUriMapping:
-              - path: http://egov-location.pucar-prod:8080/egov-location/location/v11/boundarys/_search
+              - path: http://egov-location.egov:8080/egov-location/location/v11/boundarys/_search
                 queryParam: hierarchyTypeCode=ADMIN,boundaryType=Locality,codes=$.service.address.locality.code,tenantId=$.service.tenantId
                 apiRequest: {"RequestInfo":{"apiId":"org.egov.pt","ver":"1.0","ts":1502890899493,"action":"asd","did":"4354648646","key":"xyz","msgId":"654654","requesterId":"61","authToken":"d9994555-7656-4a67-ab3a-a952a0d4dfc8","userInfo":{"id":1,"uuid":"1fec8102-0e02-4d0a-b283-cd80d5dab067","type":"EMPLOYEE","tenantId":"pb.amritsar","roles":[{"name":"Employee","code":"EMPLOYEE","tenantId":"pb.amritsar"}]}}}
```

yamllint (1.35.1)

[error] too few spaces after comma

(commas)